index/index.js

```diff
@@ -0,0 +1,108 @@
+}
+
+exports.handler = async(event, context) => {
+  const db = new AWS.DynamoDB.DocumentClient({ region: "us-east-2" });
+  
+
+  // Input for banner proxy lambda 
+  let input = {
+    FunctionName: 'arn:aws:lambda:us-east-2:741865850980:function:banner-proxy:live',
+    InvocationType: 'RequestResponse',
+    Payload: JSON.stringify(event)
+  };
+
+ // Invokes the banner lambda and gets the payload
+  const result = await invokeLambda(input);
+  
+
+  // Loops through the payload (Need to change the for loop values)
+  for (var index = parseInt(event.params.offset); index < parseInt(event.params.pageSize); index++) {
+    if(index == result.totalCount) {
+      break;
+    }
+    var len = result.data[index].meetingsFaculty.length;
+    var times = [];
+
+    for (var index2 = 0; index2 < len; index2++) {
+      // Calculates how many weeks for each course
+      var start = result.data[index].meetingsFaculty[index2].meetingTime.startDate;
+      var end = result.data[index].meetingsFaculty[index2].meetingTime.endDate;
+      var diff = weekDiff(start, end);
+      
+      // Gets faculty information for each course
+      var staff = [];
+      var len2 = result.data[index].faculty.length;
+      for(var j = 0; j < len2; j++) {
+        staff.push(result.data[index].faculty[j].displayName);
+      }
+      
+      // The meetingTimes format
+      const items = {
+        startTime: result.data[index].meetingsFaculty[index2].meetingTime.beginTime,
+        endTime: result.data[index].meetingsFaculty[index2].meetingTime.endTime,
+        building: result.data[index].meetingsFaculty[index2].meetingTime.building,
+        room: result.data[index].meetingsFaculty[index2].meetingTime.room,
+        instructors: staff,
+        monday: result.data[index].meetingsFaculty[index2].meetingTime.monday,
+        tuesday: result.data[index].meetingsFaculty[index2].meetingTime.tuesday,
+        wednesday: result.data[index].meetingsFaculty[index2].meetingTime.wednesday,
+        thursday: result.data[index].meetingsFaculty[index2].meetingTime.thursday,
+        friday: result.data[index].meetingsFaculty[index2].meetingTime.friday,
+        saturday: result.data[index].meetingsFaculty[index2].meetingTime.saturday,
+        sunday: result.data[index].meetingsFaculty[index2].meetingTime.sunday,
+        weeks: diff
+      }
+      times.push(items);
+    }
+    // The class format
+    const params = {
+      TableName: "temple-202036",
+      Item: {
+        courseName: result.data[index].courseTitle,
+        crn: parseInt(result.data[index].courseReferenceNumber),
+        isOpen: result.data[index].openSection,
+        campus: result.data[index].campusDescription,
+        attributes: result.data[index].sectionAttributes[0],
+        meetingTimes: times
+      }
+    }
+    
+  // Logs class data into the database
+  try {
+    const data = await db.put(params).promise();
+    console.log(data);
+   } catch (err) {
+    console.log(err);
+   }
+  }
+};
```

Comment:
It's generally considered bad practice to dump everything into the Lambda handler. I would try an keep the handler as concise as possible and move things into their own functions:
```
async handler (event, context) => {
sections = await invokeLambda(paramsFromEventObject);
itemsForDB = convertToDB(sections);
await putItemsInDB(itemsForDB);
}
```
Just a suggestion